src/shared/components/common/state-radios.tsx

```diff
@@ -0,0 +1,62 @@
+import { I18NextService } from "@services/index";
+import { randomStr } from "@utils/helpers";
+import classNames from "classnames";
+import { NoOptionI18nKeys } from "i18next";
+import { linkEvent } from "inferno";
+import { Component } from "inferno";
+
+export interface State {
+  value: string;
+  i18n: NoOptionI18nKeys;
+}
+
+export interface StateRadioProps {
+  className?: string;
+  allStates: State[];
```

Comment:
Don't call this `State`, that has a specific meaning in react, meaning the component state. Really this is a `RadioOption[]`
, and this component is `Radios` or `RadioButtonGroup`